modules/nf-core/biscuit/align/main.nf

```diff
@@ -25,14 +26,15 @@ process BISCUIT_ALIGN {
     def biscuit_cpus = (int) Math.max(Math.floor(task.cpus*0.9),1)
     def samtools_cpus = task.cpus-biscuit_cpus
     """
+    set -o pipefail
```

Comment:
nf-core pipelines already have this, cf https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/nextflow.config#L175

Reply:
Ok, didn't realize that - thanks!